classes.php

```diff
@@ -0,0 +1,384 @@
+		{
+			if ($health >= 90)
+			{
+				$_illness = false;
+				$_illnessName = null;
+			}
+			if ($health < 100)
+			{
+				$_health += $healRate;
+			}
+		}
+
+		#member takes damage when ill or durring event
+		public function takeDamage()
+		{
+			$_health -= $damage; 
+		}
+
+		#kills member
+		public function die()
+		{
+			$_alive = false;
+		}
+	}
+
+
+	#class for the stack or supplies avalable to the party
+	class Supplies 
+	{
+		# amount of each supply
+		public $_food;
+		public $_money;
+		public $_bait;
+		public $_clothes;
+		public $_wagonWheels;
+		public $_wagonAxle;
+		public $_wagonTongue;
+		public $_oxen;
+	
+
+		public function __construct($jobCash)
+		{
+			$_food = 0;
+			$_money = $jobCash; #Job
+			$_bait = 0;
+			$_clothes = 0;
+			$_wagonAxle = 0;
+			$_wagonWheels = 0;
+			$_wagonTongue = 0;
+
+		}
+
+		public function eat($rate)
+		{
+			$_food -= $rate;
+		}
+
+		public function setItem($ID, $amount)
+		{
+			switch ($ID):
+			case 0:
+				$_food += $amount;
+				break;
+				
+			case 1:
+				$_money += $amount;
+				break;
+
+			case 2:
+				$_bait += $amount;
+				break;
+
+			case 3:
+				$_clothes += $amount;
+				break;
+
+			case 4:
+				$_wagonWheels += $amount;
+				break;
+
+			case 5:
+				$_wagonAxle += $amount;
+				break;
+
+			case 6:
+				$_wagonTongue += $amount;
+				break;
+
+			case 7:
+				$_oxen += $amount;
+				break;
+
+
+		}
+
+	}
+
+	#class for the whole party
+	class Party
+	{
+		$_members = []; #array containing all party members
+		$_health = "Good";	#general health of the party
+		$_supplies;	#supplies class instant
+		$_livingMembers = 5; #licing members of the party
+		$_rate = 3; #rate at which food is eaten
+
+		function __construct($names, $jobCash)
+		{
+			# names are given to constructor and made into party members
+			$_members[0] = new PartyMember($names[0]);
+			$_members[1] = new PartyMember($names[1]);
+			$_members[2] = new PartyMember($names[2]);
+			$_members[3] = new PartyMember($names[3]);
+			$_members[4] = new PartyMember($names[4]);
+
+			#supplies made by passing in starting money
+			$_supplies = new Supplies($jobCash);
+
+
+		}
+
+		#set ration rate
+		function setRate($newRate)
+		{
+			$_rate = $newRate;
+		}
+
+		#food is reduced based on living party members and ration rate
+		function eat()
+		{
+			$_supplies->eat($_rate * $_livingMembers);
+		}
+
+		#checks if any members can be killed (has 0 health)
+		function killCheck()
+		{
+			foreach($_members as $body)
+			{
+				if($body->$_alive)
+				{
+					if($body->$health == 0)
+					{
+						$body->$_alive = false;
+						$_livingMembers-=1;
+					}
+				}
+			}
+
+			#code for when all members are dead
+			#ends game
+		}
+
+		#averages health and assigns a rating
+		function health()
+		{
+			$sum = 0;
+			$average = 0;
+			foreach ($_members as $body) {
+				if($body->$_alive)
+				{
+					$_sum += $body;
+				}
+			}
+
+			$average = $sum / $_livingMembers;
+
+
+			if($average => 0)
```

Comment:
This if statement will always cause health to be "game over" due to your else ifs